### PR TITLE
Update got module

### PR DESCRIPTION
**Pull request by fire015**
With the latest `got` library the whole package no longer works, so here are the relevant fixes (I have tested this on my personal extension and could successfully submit to the chrome store).

I had to update the `ava` library is it had a ton of security problems.

I am not familiar with the testing framework used. As I have done the hard-work would you mind getting the tests fixed then release to NPM?

Thanks

### CODE REVIEW

**Comment by fregante** [comment]
0.3.1 published with the previous version

**Comment by fire015** [comment]
@fregante tests now passing! Please release :)

**Comment by fregante** [comment]
Can you also change the `got` version? It was undone on `master`

File: test/fetchToken.js [hunk 1]
```diff
@@ -23,12 +23,10 @@ test.serial('Only returns token from response body', async t => {
...
-    sandbox.stub(got, 'post', (uri) => {
-        return Promise.resolve({
-            body: {
-                access_token: accessToken
-            }
-        });
+    sandbox.stub(got, 'post').returns({
+        json: sandbox.stub().returns(Promise.resolve({
+            access_token: accessToken
+        }))
```
Comment (fregante) [inline test/fetchToken.js:29]: cc @sindresorhus does this look right?

Comment (sindresorhus) [reply test/fetchToken.js:29]: LGTM

**Comment by fire015** [comment]
@fregante this is now done

**Pull request merged**